x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/FollowIndexAction.java

```diff
@@ -568,4 +598,16 @@ private static Settings filter(Settings originalSettings) {
         return settings.build();
     }
 
+    private static Map<Integer, String> convert(Map<String, String> ccrMetadata) {
```

Comment:
Should we name this method a bit less abstract? `extractIndexShardHistoryUUIDs`?